music21/roman.py

```diff
@@ -832,6 +871,17 @@ def romanNumeralFromChord(chordObj,
         'i64b3': 'Sw43',
         'io6b5b3': 'Ger65',
     }
+    secondaryDominantSwapsMajor = {
+        'II': 'V',  # II > V/V
+        'III': 'vi',  # III > V/vi
+        'VI': 'ii',  # VI > V/ii
+    }
+    secondaryDominantSwapsMinor = {
+        'I': 'iv',  # Debatable
+        'II': 'V',  # Only one shared with major
+        'III': 'VI',  # NB upper (VI)
+        'IV': 'VII',  # sic, no flat (bVII) by default
```

Comment:
Thanks. Yes this relates to the m21 convention for 6th / 7th degree handling by `quality` which (as you know) sets `VII` on the natural, rather than harmonic minor seventh (e.g. Bb rather than B in c). This is consistent within m21 (or at least it should be!) but is not with all other code libraries / schools of thought -- hence the reminder. Perhaps 'no flat needed' is clearer?